enterprise/frontend/src/embedding/data-picker/DataSelector/DataSelector.jsx

```diff
@@ -366,16 +369,29 @@ export class UnconnectedDataSelector extends Component {
     } else if (
       // Schema id is explicitly set when going through the New > Question/Model flow,
       // whereas we have to obtain it from the state when opening a saved question.
-      (this.state.selectedSchemaId || this.state.selectedSchema?.id) &&
-      steps.includes(TABLE_STEP)
+      this.state.selectedSchemaId ||
+      this.state.selectedSchema?.id
```

Comment:
`steps.includes(TABLE_STEP)` is removed because it's always true in embedding.

https://github.com/metabase/metabase/blob/f7a24711c48f957636ec6e5af0e37a0a9df20cc3/enterprise/frontend/src/embedding/data-picker/DataSelector/DataSelector.jsx#L54